scripts/buidl.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+# Script for building a host with injected json data
+
+set -o pipefail
+trap cleanup EXIT
+trap cleanup SIGINT
+
+data_nix="/tmp/data.nix"
+
+# Cleanup which will be executed at exit
+cleanup() {
+  rm -rf "$data_nix"
```

Comment:
Sure, but remember that this could potentially contain very sensitive information.

Reply:
I see, can we remove recursive and force? Seems like a hazard to me.

Reply:
I think that flags can be removed. By the way, I missed the main reason why instant cleanup has to be there. You may have already noticed it, but I'll explain it for the sake of documentation:

https://github.com/ponkila/nixobolus/blob/7249440cf18d45ae7c1cfe77d076bc052457fd36/flake.nix#L238

We have to ensure that the `data.nix` file is only imported into the host configuration when wanted. So, if we were to build a host with injected JSON data and later decide to build the base configuration on the same machine *without* the injected data, it becomes crucial to ensure that the `data.nix` file does not exist. 

This situation is also potentially dangerous as it could enable the malicious or accidental injection of configuration data without the user's knowledge. We should keep this in mind for the future and consider measures to prevent such events from happening.